### DISCUSSION TITLE
Use BADA for performance calculation

### DISCUSSION THREAD

**Discussion by GunnarQuante** [contrailcirrus/pycontrails#73] [Q&A] [answered]
Hi all, 

at first, thanks a lot for making this model package openly available! I have been working on with it for some time now and would like to use BADA instead of the PS-model for performance calculations. I have a BADA 3 license and the performance files. 

When running pip install "pycontrails-bada @ git+ssh://git@github.com/contrailcirrus/pycontrails-bada.git" I get the message "The system cannot find the file". I suppose the reason is that the extension is private due to license restrictions (https://py.contrails.org/install.html#bada) Is that correct? And if so, is there a way of enabling the use of BADA for users with a BADA license? 

Thanks a lot!

**Comment by zebengberg** [comment, marked as answer]
Hi @GunnarQuante 

Please email help@contrails.org with this request.

Thanks!

**Reply by GunnarQuante** [reply]
Hi @zebengberg - thanks a lot, I will do so :)
